admin/modules/user/users.php

```diff
@@ -1526,8 +1542,24 @@ function toggleAction()
 		$mybb->input['moderateposting'] = 1;
 		if($user['moderationtime'] != 0)
 		{
-			$expired = my_date('relative', $user['moderationtime'], '', 2);
-			$existing_info = $lang->sprintf($lang->moderate_length, $expired);
+			$remaining = $user['moderationtime']-TIME_NOW;
+			$expired = nice_time($remaining, array('seconds' => false));
+
+			$color = 'inherit';
+			if($remaining <= 0 || $remaining < 3600)
```

Comment:
What's the point of checking `$remaining <= 0`? If it's lower than 3600 it's also lower than 0.